doc/source/tutorial/stats/sampling.md

```diff
@@ -188,7 +190,11 @@ In the above example, we have set up an object of the
 standard normal distribution. Now, we can start sampling from our
 distribution by calling the `rvs` method:
 
-```python
+```{eval-rst}
+.. try_examples::
+
+    >>> from scipy.stats.sampling import TransformedDensityRejection
+    >>> rng = TransformedDensityRejection(dist, domain=(-1, 1), random_state=urng)
```

Comment:
Some of these import statements will be required in these code snippets to make sure that the button works and the examples run without issues for users. I'm not sure if there is a better or cleaner way to do this, because it will create repeated imports (though running them won't be a problem because imports are cached)